src/sentry/api/endpoints/organization_pinned_searches.py

```diff
@@ -30,7 +30,7 @@ def validate_type(self, value):
 
 @region_silo_endpoint
 class OrganizationPinnedSearchEndpoint(OrganizationEndpoint):
-    owner = ApiOwner.OWNERS_SNUBA
+    owner = ApiOwner.UNOWNED
```

Comment:
These were assigned to SSS just because there was a word `search` in them. Should be unowned.